data/user_list.json

```diff
@@ -0,0 +1,12 @@
+{
+  "users": [
+    {
+      "username": "abcde",
+      "password": "11111"
```

Comment:
We normally store hash values of password, not the password itself. Let's just have a look at some hash functions like md5.

Reply:
Okay